### ISSUE TITLE
Fill symbolizer affects line symbolizer (and vice versa)

### ISSUE THREAD

**Issue by jmacura** [hslayers/hslayers-ng#2148]
# Bug

**Describe the bug**
When a Line symbolizer is set, it also affects the symbology of polygon boundaries. Vice versa, when a Fill symbolizer is set with some outline colour, it affects the symbology of lines.

**To Reproduce**
1. Open some layer with polygons and lines or draw some.
2. Go to symbolize layer
3. Create Line symbolizer
4. Set some colour and width

**Expected behavior**
Line symbolizer shall be only applied to lines. Fill symbolizer shall be only applied to polygons.

**Screenshots**
![image](https://user-images.githubusercontent.com/5598693/132926342-321d573d-3edb-4599-a527-ac9472ce2a02.png)
![image](https://user-images.githubusercontent.com/5598693/132926360-e8b6612b-b8fc-4a96-a96a-81c06b0fa969.png)
![image](https://user-images.githubusercontent.com/5598693/132926369-0a53a6bd-ba19-4cb4-a6fd-0ef84d768c5d.png)

**Additional context**
Or perhaps not... Could this be a desired behaviour?

**Comment by raitisbe** [comment]
Yeah, I dont think we can classify it as a bug. User must know what he's doing and probably the fix would need to be made in the geostyler anyway. Will leave it open for a bit.

**Comment by fzadrazil** [comment]
What is misleading here is the fact that there is an outline settings in the Fill symbolizer and line settings in Line symbolizer which does the same, but are not synchronized. So the user would suppose Outline only affects fill symbols and Line symbolizer only polylines. Eg. when you change the outline color, it won't change the value in Line symbolizer, but the lines get changed and vice versa.

**Comment by raitisbe** [comment]
Yes, but looking from the other side, there would not be a possibility to draw these nice contours around the line in another color, if we synchronized fill symbolizers outline color to line symbolizers color.

**Comment by jmacura** [comment]
Just leaving a note that this will be confusing for any end-user and I still consider it a UX-bug.
![obrazek](https://user-images.githubusercontent.com/5598693/156794117-3067bef7-db02-4fe9-88db-cbb090aa95e2.png)
